lib/api/kuzzle.js

```diff
@@ -128,9 +128,31 @@ class Kuzzle extends EventEmitter {
       .then(() => this.indexCache.init())
       .then(() => this.repositories.init())
       .then(() => this.funnel.init())
+      .then(() => {
+        if (! params.mappings) {
+          return null;
+        }
+
+        return this.janitor.loadMappings(params.mappings);
```

Comment:
(nitpicking) Wouldn't be cleaner if the `janitor.loadXxx` methods had a defined behavior if no mappings are provided to them?

This way you could simply write: `then(() => this.janitor.loadMappings(params.mappings))`

Reply:
I leave the `loadSecurities` to keep the message about the loading time